### PR TITLE
Log success of retrieving chrome extension banner configs

### PR DESCRIPTION
**Pull request by horymury**
@saghul @damencho

### CODE REVIEW

File: react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js [hunk 1]
```diff
@@ -102,11 +102,25 @@ class ChromeExtensionBanner extends PureComponent<Props, State> {
...
      */
-    async componentDidUpdate() {
+    async componentDidUpdate(prevProps) {
         if (!this._isSupportedEnvironment()) {
             return;
         }
 
+        if (
+            this.props.chromeExtensionUrl
+            && !prevProps.chromeExtensionUrl
```
Comment (saghul) [inline react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js]: doesn't this fit in one line? note that our eslint config checks for 120 line lengths

File: react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js [hunk 2]
```diff
@@ -102,11 +102,25 @@ class ChromeExtensionBanner extends PureComponent<Props, State> {
...
+        ) {
+            logger.info('Chrome extension URL found.');
+        }
+
+        if (
+            this.props.chromeExtensionsInfo.length
+            && !prevProps.chromeExtensionsInfo.length
```
Comment (saghul) [inline react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js]: ditto

File: react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js (hunk 1 above)
Comment (horymury) [reply react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js]: done

File: react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js (hunk 2 above)
Comment (horymury) [reply react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js]: done

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by saghul** [comment]
Jenkins please test this please.

**Pull request merged**